lib/msal-core/test/UserAgentApplication.spec.ts

```diff
@@ -28,6 +28,7 @@ import { ServerRequestParameters } from "../src/ServerRequestParameters";
 import { TEST_URIS, TEST_DATA_CLIENT_INFO, TEST_HASHES, TEST_TOKENS, TEST_CONFIG, TEST_TOKEN_LIFETIMES } from "./TestConstants";
 import { IdToken } from "../src/IdToken";
 import { TimeUtils } from "../src/utils/TimeUtils";
+import { request } from "http";
```

Comment:
did you mean to import this ?

Reply:
oops, thanks. I think it is vscode being "smart"

Reply:
ok approved when its removed, feel free to merge

Reply:
Done. Please check.